src/vm_id.rs

```diff
@@ -57,6 +57,11 @@ impl VmIdTracker {
         self.current_version
     }
 
+    /// Returns the VMID length.
+    pub fn get_vmid_len(&self) -> u64 {
+        *self.vmid_bits.call_once(get_vmid_bits)
+    }
```

Comment:
Should we just put the # of VMID bits in a global `Once`?

Reply:
Makes sense given it shouldn't change between cores. I will change this in next push.